CODEOWNERS

```diff
@@ -411,6 +411,7 @@ build.json @home-assistant/supervisor
 /homeassistant/components/fortios/ @kimfrellsen
 /homeassistant/components/foscam/ @skgsergio
 /tests/components/foscam/ @skgsergio
+/homeassistant/components/frank_energie/ @bajansen @DCSBL
```

Comment:
Please run hassfest as it should also add you to the tests as code owner

Reply:
hassfest did not do this for me, but manually added it now

Reply:
Now hassfest rejects.. 🤷‍♂️

Reply:
That's interesting